.github/CODEOWNERS

```diff
@@ -12,6 +12,7 @@
 #/avm/ptn/avd-lza/management-plane/ @Azure/avm-ptn-avd-lza-managementplane-module-owners-bicep @Azure/avm-module-reviewers-bicep
 #/avm/ptn/avd-lza/networking/ @Azure/avm-ptn-avd-lza-networking-module-owners-bicep @Azure/avm-module-reviewers-bicep
 #/avm/ptn/avd-lza/session-hosts/ @Azure/avm-ptn-avd-lza-sessionhosts-module-owners-bicep @Azure/avm-module-reviewers-bicep
+/avm/ptn/azd/acr-container-app/ @Azure/avm-ptn-azd-acrcontainerapp-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Comment:
team does not exist, or not in the correct hierachy

Reply:
Hey @NanaXiong00, it's about this [spec](https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships) in the contribution guide.
Adding the correct parent team to your teams ensures you have the permissions in the future to approve PRs for your module

Reply:
Seems to be in order now 💪